dev/oracle/latest/start.sh

```
@@ -0,0 +1,18 @@
+# Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved
+
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)